packages/vue-apollo-composable/src/util/loadingTracking.ts

```diff
@@ -11,7 +11,8 @@ export interface AppLoadingTracking extends LoadingTracking {
 }
 
 export function getAppTracking () {
-  const root: any = getCurrentInstance().$root
+  const vm = getCurrentInstance()
```

Comment:
@frandiox Type checkin fails, you probably need `const vm: any = getCurrentInstance()`

Reply:
@reegodev Thanks for noticing!